auth-server/src/main/java/com/epam/petclinic/authentication/WebSecurityConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.petclinic.authentication;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Override
+    @Bean
+    public AuthenticationManager authenticationManagerBean() throws Exception {
```

Comment:
Yes, We need to do it to create bean and inject object to AuthorizationServerConfig class.